flang/tools/bbc/bbc.cpp

```diff
@@ -287,7 +294,7 @@ int main(int argc, char **argv) {
   Fortran::parser::AllSources allSources;
   Fortran::semantics::SemanticsContext semanticsContext{
       defaultKinds, options.features, allSources};
-  semanticsContext.set_moduleDirectory(moduleDirs.front())
+  semanticsContext.set_moduleDirectory(moduleDir)
```

Comment:
OK. But I wonder why we don't have the ability to specify a set of directories for module files, just like a set of include directories. That seems weird.

Reply:
Because -module is for output, so having a single directory make sense.  It also works for input, as a convenience, on PGI -- I don't know about f18.   The -I option is the general-purpose option for finding modules.

Reply:
Oh, I see. This changes what `-module` means.

Is there a reason `-intrinsic-module-directory` is special? It still seems like that should allow more than one directory, at least in a "real" driver.

Reply:
Fortran intrinsic modules are special. These are the modules defined by the language.  F18 decided to designate a directory to hold the intrinsic modules.  I don't know if we'll need another.